src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <div className="user">
+      <p className="user__name">
+        {user !== null ? user.name : 'Unknown name'}
+      </p>
+      <p className="user__email">
+        {user !== null ? user.email : 'Unknown email'}
+      </p>
+    </div>
+  );
+};
```

Comment:
You render this component if user is not null, inner checks (user !== null) are redundant here